.circleci/test-github-repair.sh

```diff
@@ -51,7 +51,7 @@ createTestPR 'test-after-repair' "Test after repair"
 # Create a bunch of pull requests that will not run any tests.
 # We do this so that we'll have to make more than one API request
 # to find our test PR.
-for n in $(seq 1 10) ; do
+for n in $(seq 1 5) ; do
```

Comment:
Is five PRs enough to require a second API request to find the test PR? I thought that the API by default paged after ten items. If we're not forcing the second API call, then we might as well have zero base PRs.

Reply:
It looks like this happens in the test for build:project:repair; that whole section takes just under a minute. Maybe it isn't so costly to just make ten PRs?

Reply:
I was trying to reduce API requests to try to avoid hitting Github limit

Reply:
Just re-read and understood this. I'll go back to 10